packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/entities/Event.java

```diff
@@ -57,12 +59,13 @@ public class Event extends PanacheEntityBase implements Serializable {
 
 	/**
 	 * Gets events by type and an option search string for values
-	 * @param type				the {@link EventType}
-	 * @param projectId		the optional project id
-	 * @param username		the username of the current user
-	 * @param like				optional search string for values of matching events
-	 * @param limit				the maximum number of events to return
-	 * @return						a list of {@link Event} matching the input
+	 *
+	 * @param type      the {@link EventType}
+	 * @param projectId the optional project id
+	 * @param username  the username of the current user
+	 * @param like      optional search string for values of matching events
+	 * @param limit     the maximum number of events to return
+	 * @return a list of {@link Event} matching the input
```

Comment:
@dvince2 and @chris-dickson you need to decide how your Java Doc is setup.
Every other PR there is a change just for that.

Reply:
@chris-dickson I'm using the default settings from IntelliJ